src/librustdoc/html/markdown.rs

```diff
@@ -78,6 +80,7 @@ pub struct Markdown<'a>(
     /// Default edition to use when parsing doctests (to add a `fn main`).
     pub Edition,
     pub &'a Option<Playground>,
+    pub u32,
```

Comment:
Please add a doc comment explaining what this field is. Out of context, it's impossible to know what it is without going through the code.